tla-io/src/main/scala/at/forsyte/apalache/tla/imp/SanyImporter.scala

```diff
@@ -25,16 +25,31 @@ class SanyImporter(sourceStore: SourceStore, annotationStore: AnnotationStore) {
    *
    * @param file
    *   an input file
+   * @param useLibraryPaths
+   *   Whether to use [[file]]'s containing directory as library path. Disable this flag when you [[loadFromSource]]; we
+   *   then look up names in the paths given by the `TLA-Library` Java system variable instead.
```

Comment:
I find this name confusing. Shall we call it like `useRelativePathLookup`?

Reply:
Aaaah, I was looking for a good name 😄 Thanks!
Changed in 6e63be6f